lib/common.sh

```diff
@@ -111,6 +111,7 @@ export DEFAULT_HOSTS_MEMORY=${DEFAULT_HOSTS_MEMORY:-8192}
 export CLUSTER_NAME=${CLUSTER_NAME:-"test1"}
 export CLUSTER_APIENDPOINT_IP=${CLUSTER_APIENDPOINT_IP:-"192.168.111.249"}
 export KUBERNETES_VERSION=${KUBERNETES_VERSION:-"v1.18.0"}
+export K3_BINARIES_VERSION=${K3_BINARIES_VERSION:-"1.18.0"}
```

Comment:
why not to use KUBERNETES_VERSION variable ?

Reply:
They are not in the same format. (v)
:-"v1.18.0"
:-"1.18.0"

Reply:
I see, but then you should change also K3_BINARIES_VERSION to be `default('1.18.0', true) }}"` in vm-setup/roles/v1aX_integration_test/vars/main.yml

Reply:
you could make KUBERNETES_VERSION to be 1.18.0 and then wherever used add the v in front, that would be better than creating this duplicated variable